frontend/packages/ux-editor/src/components/config/EditFormComponent.tsx

```diff
@@ -60,60 +52,36 @@ export const EditFormComponent = ({
     return componentSpecificEditConfig[component.type];
   };
 
-  const toggleShowBetaFunc = (event: React.ChangeEvent<HTMLInputElement>) => {
-    setShowComponentConfigBeta(event.target.checked);
-    // Ensure choice of feature toggling is persisted in local storage
-    if (event.target.checked) {
-      addFeatureFlagToLocalStorage('componentConfigBeta');
-    } else {
-      removeFeatureFlagFromLocalStorage('componentConfigBeta');
-    }
-  };
-
   const isUnknownInternalComponent: boolean = !formItemConfig;
   if (isUnknownInternalComponent) {
     return <UnknownComponentAlert componentName={component.type} />;
   }
 
   return (
     <Fieldset className={classes.root} legend=''>
-      <FormField
-        id={component.id}
-        value={showComponentConfigBeta || false}
-        onChange={toggleShowBetaFunc}
-        propertyPath={formItemConfig.propertyPath}
-        componentType={component.type}
-        helpText={t('ux_editor.edit_component.show_beta_func_help_text')}
-        renderField={({ fieldProps }) => (
-          <Switch {...fieldProps} checked={fieldProps.value} size='small'>
-            {t('ux_editor.edit_component.show_beta_func')}
-          </Switch>
-        )}
-      />
-      {showComponentConfigBeta && isPending && (
+      {isPending && (
         <StudioSpinner
           showSpinnerTitle
           spinnerTitle={t('ux_editor.edit_component.loading_schema')}
         />
       )}
-      {showComponentConfigBeta && !isPending && (
+      {!isPending && (
         <FormComponentConfig
           schema={schema}
           component={component}
           editFormId={editFormId}
           handleComponentUpdate={handleComponentUpdate}
         />
       )}
-      {!showComponentConfigBeta && (
-        <>
-          {renderFromComponentSpecificDefinition(getConfigDefinitionForComponent())}
-          <ComponentSpecificContent
-            component={component}
-            handleComponentChange={handleComponentUpdate}
-            layoutName={selectedLayout}
-          />
-        </>
-      )}
+
+      <>
+        {renderFromComponentSpecificDefinition(getConfigDefinitionForComponent())}
```

Comment:
Since there are no updates in this PR of the custom config for the components, we run the risk of double up configuration for some of the components. We should find out which components can keep the custom-config and which components all properties are covered by the auto-generated config - for the latter we can remove the custom config.